pyne/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+#unset LDFLAGS
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export CMAKE_LIBRARY_PATH="$PREFIX/lib:$CMAKE_LIBRARY_PATH"
+export PATH="$PREFIX/bin:$PATH"
+export DYLD_FALLBACK_LIBRARY_PATH="$PREFIX/lib/cyclus:$PREFIX/lib"
+UNAME=$(uname)
+
+if [[ "${UNAME}" == 'Linux' ]]; then
+  ln -s $PREFIX/lib/libhdf5.so.9 $PREFIX/lib/libhdf5.so.8
+  ln -s $PREFIX/lib/libhdf5_hl.so.9 $PREFIX/lib/libhdf5_hl.so.8
+else
+  ln -s $PREFIX/lib/libhdf5.9.dylib $PREFIX/lib/libhdf5.8.dylib
+  ln -s $PREFIX/lib/libhdf5_hl.9.dylib $PREFIX/lib/libhdf5_hl.8.dylib
+  export CC=gcc
```

Comment:
This line isn't relevant to CI since we are only launching Linux jobs.  Also, I don't think that PyNE can be built with clang - at least not fully - due to fortran linking issues.